src/main/java/gighub/worketserver/domain/ContractModify.java

```diff
@@ -0,0 +1,40 @@
+package gighub.worketserver.domain;
+
+import gighub.worketserver.domain.constants.ModifyStatus;
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "contract_modify")
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+@Builder
+public class ContractModify {
```

Comment:
테이블은 동사 기반 네이밍보다 명사형을 기반으로 해서 도메인 의미를 명확히 드러내는 이름이 좋을 것 같아요!
```suggestion
public class ContractModificationRequest {
```

Reply:
넵 !! ContractModificastion으로 수정하겠습니다!